### ISSUE TITLE
Show all errors reported by axe

### ISSUE THREAD

**Issue by kborchers** [microsoft/accessibility-insights-web#1245]
## Is your feature request related to a problem? Please describe.

Running a Fast Pass scan will come back as passing even if there are some color contrast issues. This could potentially happen with other missed errors but I noticed it with contrast specifically. It appears that since axe will occasionally accompany an error report with a message that it couldn't determine the background or foreground color or the text content is too short or similar, Accessibility Insights is throwing out those issues.

## Describe the desired outcome

It would be great if Accessibility Insights would still display those errors along with the accompanying message to allow users to determine if the issue needs to be addressed or not.

## Describe alternatives you've considered

Currently my only alternative is to run a Fast Pass and even if it comes back passing I have to then also run axe independently to ensure no issues were missed.

**Comment by ferBonnin** [comment]
Hi @kborchers thanks for your feedback! In FastPass, we only show issues that are WCAG violations (and not best practice, needs review or experimental rules).

For color contrast in particular, when axe-core returns _needs review_, we show this results in Assessment, in the Text legibility test >  Contrast requirement, and ask the user to investigate if its a failure or not.

**Comment by kborchers** [comment]
IMO, it seems like anything with "needs review" should still be surfaced in FastPass because now that I know FastPass is hiding actual issues, FastPass is no longer useful to me and I can only ever use Assessment or just use axe directly.

**Comment by ferBonnin** [comment]
We understand the concern that there are more things you need to take a look at even after a FastPass, but that would be the case even if we showed _needs review_ items in FastPass.

The goal of FastPass is not to be complete, but to help you check for common accessibility issues in **less than 5 minutes**, thus we only show things that we know are failures and add the assisted manual test for tabstops. For full WCAG 2.1 AA coverage, you need to do an Assessment.

That being said, I want to reiterate we appreciate the feedback and point of view and that it is not off the table to do more explorations around this in the future.

**Comment by DaveTryon** [comment]
@kborchers, some checks (like the color contrast checks) have been included in the **Needs Review** FastPass tab. That will hopefully help to some extent. Including all of the axe results, though, if out of scope for our current priorities.

Thanks for using Accessibility Insights!
